db/db_compaction_test.cc

```diff
@@ -601,7 +601,8 @@ TEST_F(DBCompactionTest, ZeroSeqIdCompaction) {
   CompactionOptions compact_opt;
   compact_opt.compression = kNoCompression;
   compact_opt.output_file_size_limit = 4096;
-  const int key_len = compact_opt.output_file_size_limit / 5;
+  const size_t key_len =
+    static_cast<size_t>(compact_opt.output_file_size_limit) / 5;
```

Comment:
The problem with this patch is that it assumes that size_t is always 64-bit and it is not. It changes with the build whereas output_file_size_limit is always 64-bit and that is why the cast is there because there is a potential loss of data when size_t is 32-bit just like int. So this change does not improve anything.